packages/kit/src/runtime/server/index.js

```diff
@@ -315,7 +315,11 @@ export async function respond(request, options, state) {
 
 				// we can't load the endpoint from our own manifest,
 				// so we need to make an actual HTTP request
-				return await fetch(request);
+				try {
+					return await fetch(request);
+				} catch (e) {
+					return new Response(coalesce_to_error(e).message, { status: 500 });
+				}
```

Comment:
By try-catching this, we prevent `handleError` from being called, which seems bad (though honestly I'm not sure how this _could_ fail — `fetch` only throws if there's a network (including CORS errors) which shouldn't really apply on the server. Is there a reason for this change?

Reply:
I was just being paranoid, looking at all code paths and seing if anything could theoretically fail, and fetch is such a code path. In the related issue it read "handle should _never_ fail" so I thought "ok let's try that then". Maybe it's easier to wrap the whole code into one more big try-catch.

Reply:
That's what I was thinking, yeah